Gemfile

```diff
@@ -19,6 +19,8 @@ gem "sidekiq", "~> 6.0"
 gem "stimulus_reflex", "~> 2.1"
 gem "valid_email", "~> 0.1"
 gem "webpacker", "~> 4.0"
+gem "kaminari", "~> 1.1.1"
```

Comment:
I propose that we switch to the pagy gem as it's more performant. https://github.com/ddnexus/pagy

Reply:
First I’ve ever heard of it!